README.md

```diff
@@ -46,73 +46,29 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
-   ```
-
-5. Build the ForgeRock JavaScript SDK:
-
-   ```
-   npm run build
-   ```
-
-6. Open `samples/embedded-login/index.html` and edit the configuration values to match your AM instance.
-
-7. Serve the `samples` directory by using a simple HTTP server.
-
-   - Run `npm run start:samples`
-
-8. Edit your `/etc/hosts` file to point your localhost (e.g. `127.0.0.1`) to `sdkapp.example.com`
-
-9. In a [supported web browser](#requirements), navigate to `https://sdkapp.example.com:8443`, and then click **Embedded Login**.
-
-<!------------------------------------------------------------------------------------------------------------------------------------>
-<!-- SAMPLES - List the samples we include with the SDKs, where they are, briefly what they show. -->
-
 ## Samples
 
 ForgeRock provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of ForgeRock and are not officially supported.
 
-- **Embedded login - `/samples/embedded-login`**
+To try out the ForgeRock JavaScript SDK please look at one of our samples:
+
+- [**Embedded login - `/samples/embedded-login`**](./samples/embedded-login/README.md)
 
   In most real-world scenarios, you will want to have full control over the UI. In these cases, you can use `FRAuth` to obtain typed callback instances from authentication trees and render the UI in whatever way makes sense for your application.
 
-  You can run this sample app with the `npm run start:samples` command. Please [see the Getting Started instructions](#getting-started) for more details.
+  Please [see the Getting Started instructions](./samples/embedded-login/README.md#getting-started) for more details.
 
-- **Central login - `/samples/central-login`**
+- [**Central login - `/samples/central-login`**](./samples/embedded-login/README.md)
 
   In cases in which a centralized login application is desired, in contrast to an embedded login form, the SDK provides a method for leveraging an OAuth flow known as Authorization Code Flow (with PKCE). By using an option (login of "redirect") provided by `TokenManager` to request OAuth/OIDC tokens, when the user is not authenticated, the SDK will redirect the user to the configured login application with the ForgeRock platform. Once authenticated, the user will be redirected back to the original application to attain OAuth/OIDC tokens to complete the user flow.
 
-  You can run this sample app with the `npm run start:samples` command. Please [see the Getting Started instructions](#getting-started) for more details.
+  Please [see the Getting Started instructions](./samples/central-login/README.md#getting-started) for more details.
 
 - **React todos - `/samples/reactjs-todos`**
 
   A sample React JS application is provided to demonstrate how the JavaScript SDK can be implemented within a React context. You can read more about this application by [visiting its README.md file](samples/reactjs-todo/README.md).
```

Comment:
Some of this text is duplicated between this README and the nested READMEs in the samples. Should we remove these descriptions and just have the links?